docs/GOVERNANCE.md

```diff
@@ -5,6 +5,16 @@
 This document describes the governance process under which the CloudEvents
 project will manage this repository.
 
+For easy reference, additional documentation related to how this project
+(and it's subprojects) operate are listed below:
```

Comment:
it's => its

Reply:
(And I'd probably remove the parentheses - otherwise if you ignore the parenthetical part, you get a singular/plural mismatch.

Reply:
done